Site/ASAB Maestro/Descriptors/elasticsearch.yaml

```diff
@@ -29,7 +29,7 @@ descriptor:
     http.port: null  # Filled in by the Remote Control (ie 9200)
     transport.port: null  # Filled in by the Remote Control (ie 9300); 
 
-    ES_JAVA_OPTS: "-Dlog4j2.formatMsgNoLookups=true"  # Remote Control sets by default "-Xms2g -Xmx2g" for master node and "-Xms28g -Xmx28g" for data nodes
+    ES_JAVA_OPTS: null  # Remote Control sets by default "-Xms2g -Xmx2g" for master node and "-Xms28g -Xmx28g" for data nodes
```

Comment:
Remote control can make even more "educated" decision about the amount of the memory for ES - based on our yesterday experiment on VM.
It means that RC will have an access to the "memory size" of the instance; from the model or telemetry (this is indeed future development).

Reply:
I'll save this note, but it won't happen any soon, I'm afraid